nixos/modules/hardware/tuxedo-keyboard.nix

```diff
@@ -0,0 +1,37 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let 
+  cfg = config.tuxedo-keyboard;
+  tuxedo-keyboard = config.boot.kernelPackages.tuxedo-keyboard;
+in
+  {
+    options.tuxedo-keyboard = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+
+        description = ''
+          Enables the tuxedo-keyboard driver.
+          
+          To configure the driver, pass the options to the `boot.kernelParams` configuration.
```

Comment:
The docs aren't rendered with markdown, but docbook xml instead. So it's `<literal> boot.kernelParams</literal>` and `<link xlink:href="https://example.com"/>`. Render the docs with `nix-build nixos/release.nix -A manualHTML.x86_64-linux` and check the result to see whether it rendered decently.

Reply:
Thanks for pointing me there. I wasn't aware how the docs were made :) Wdyt about this? 9d818dc4846a1f779dddf000c6ef9aa54916d2e5